### DISCUSSION TITLE
BitOps Main site FQDN

### DISCUSSION THREAD

**Discussion by mickmcgrath13** [bitovi/bitops#243] [Ideas]
should we get `bitops.sh` ?

I love the idea of bitops.sh (it’s $76).  It’s also misleading, though, because it’s a docker image and not a script.  Helm, for example, has https://helm.sh/, but it’s a CLI command, so it’s more appropriate..

**Comment by mickmcgrath13** [comment]
bitops.com is not available...

whois: https://www.whois.com/whois/bitops.com

could/should we reach out to see if we can buy it?

**Comment by mickmcgrath13** [comment]
`https://bit.ops` would be great, but `.ops` doesn't exist.

Could we make/request it?
- [standard request process](https://newgtlds.icann.org/en/applicants/global-support/faqs/faqs-en) is complex and extremely expensive
  - you essentially need to build and maintain the infrastructure to manage registrants for the tld (i.e. operate as a registry)...
  - more info here: https://serverfault.com/questions/243316/can-i-create-a-top-level-tld-michael-for-instance
- maybe could request `.ops` from another company who operates as a registry already (like [donuts inc](https://donuts.domains/))

**Comment by mickmcgrath13** [comment]
Other options:
- bitops.consulting
- bitops.technology
- bitops.software
- bitops.cloud

**Reply by bmomberger-bitovi** [reply]
bito.ps is also an option.

**Comment by bmomberger-bitovi** [comment]
> It’s also misleading, though, because it’s a docker image and not a script.

platform.sh is also a PaaS and not a script, but that doesn't seem to have hurt them.

**Comment by arm4b** [comment]
* I wouldn't imagine the ~`bitops.com`~ to be available as it was first registered in 2008 and .com is always hard to get.
* For me, the preferred ~`bitops.dev`~ and ~`bitops.io`~ are already taken.
* There is also `bitops.ci`, but no ~`bitops.cd`~ which is more relevant to the project.
* `bito.ps` is not bad, looks great! A note it's a Palestine domain and might be harder to get and administer. Worth exploring.
* `bitops.sh` looks good so far too!

**Reply by mickmcgrath13** [reply]
`.ci` is for the ivory coast (Côte d'Ivoire)

**Comment by mickmcgrath13** [comment]
I purchased `bitops.sh`

**Reply by mickmcgrath13** [reply]
thanks @bmomberger-bitovi and @armab.

**Comment by mickmcgrath13** [comment]
I'd still like to keep our eye on bitops.com. I set a reminder to check again in case it's not auto-renewed.  Otherwise, we can see about contacting the owner.

**Comment by mickmcgrath13** [comment]
set up simple domain forwarding to bitops docs for now at: http://bitops.sh

**Comment by mickmcgrath13** [comment]
https://bitops.sh is complete.

BitOps.com is still unavailable...

Locking this thread
